DuckDuckGo/TabBar/View/TabBarViewController.swift

```diff
@@ -703,7 +703,12 @@ extension TabBarViewController: TabCollectionViewModelDelegate {
         updateEmptyTabArea()
         hideTabPreview()
         if tabMode == .overflow {
-            collectionView.scroll(to: IndexPath(item: index))
+            let isLastItem = collectionView.numberOfItems(inSection: 0) == index + 1
```

Comment:
I tried a couple of different solutions and investigated why `scroll(to:` doesn't scroll to the item completely. Unfortunately, I haven't found a reason so decided to create this simple workaround to fix the most common problem